hedera-node/hapi/build.gradle.kts

```diff
@@ -25,7 +25,7 @@ description = "Hedera API"
 
 // Add downloaded HAPI repo protobuf files into build directory and add to sources to build them
 tasks.cloneHederaProtobufs {
-    branchOrTag = "add-pbj-types-for-state"
+    branchOrTag = "main"
```

Comment:
I agree with @netopyr that this is a dangerous thing to do. But, while the 'tag' part of this doesn't seem to work...

Reply:
Let's fix this as soon as the build is fixed and tags are allowed again.